src/access/profiling/manager.py

```diff
@@ -47,6 +48,23 @@ def __init__(self, work_dir: Path, archive_dir: Path):
                     logger.info(f"Found archived experiment: {branch_name}")
                     self.experiments[branch_name] = ProfilingExperiment(branch_path)
 
+    def __repr__(self) -> str:
+        """Returns a string representation of the ProfilingManager."""
+        summary = f"<{type(self).__name__}>\n"
+        summary += f"    Working directory: {self.work_dir!r}\n"
+        summary += f"    Archive directory: {self.archive_dir!r}\n"
+        summary += "    Experiments:\n"
+        for name, exp in self.experiments.items():
+            summary += f"        '{name}': {exp!r}\n"
+        summary += "    Data:\n"
+        if self.data == {}:
+            summary += "        No parsed data.\n"
+        else:
+            for name, ds in self.data.items():
+                summary += f"        '{name}':\n"
+                summary += textwrap.indent(f"{ds}\n", "            ")
```

Comment:
Is the second argument intended to align the text? If that amount of white-space should be the same across the entire printed outputs - may be assign to a variable (say, `prefix_wspace`) and use that throughout the entire method?